modules/ducktests/tests/certs/functions.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+function makeRoot() {
+    ALIAS=$1
+    DNAME=$2
+    PSWD=$3
+
+    if [[ ${ALIAS} == "" ]] || [[ ${DNAME} == "" ]] || [[ ${PSWD} == "" ]]
+    then
+        error "makeRoot: Need ALIAS, DNAME, PSWD"
+    fi
+
+    rm ${ALIAS}.jks
+    rm ${ALIAS}.pem
+
+    keytool -genkeypair -keystore ${ALIAS}.jks -alias ${ALIAS} -dname "${DNAME}" -ext bc:c -storepass ${PSWD} \
```

Comment:
Please check bash checkstyle with Inteliij IDEA plugin. It will write down all possible issues. At least ${PSWD} must be in quotes, as it can contain special symbols like &.

Reply:
fixed